ops/model.py

```diff
@@ -2798,16 +2798,17 @@ def stop_checks(self, *check_names: str) -> list[str]:
             raise TypeError('stop-checks expected at least 1 argument, got 0')
 
         stopped_checks = self._pebble.stop_checks(check_names)
-        description = (
-            f'Asked to stop checks {",".join(check_names)} - '
-            f'stopped checks {",".join(stopped_checks)}'
-        )
-        _log_security_event(
-            'WARN',
-            _SecurityEvent.SYS_MONITOR_DISABLED,
-            f'{os.getuid()},{",".join(check_names)}',
-            description=description,
-        )
+        for check in check_names:
+            if check in stopped_checks:
+                description = f'Stopped check {check}'
+            else:
+                description = f'Failed to stop check {check}'
```

Comment:
Note that it may not be a failure, but just that the check is already stopped. As such, I wouldn't log anything in this case.